tests/Data/Entries/EntryQueryBuilderTest.php

```diff
@@ -688,4 +688,42 @@ public function entries_are_found_using_offset()
         $this->assertCount(2, $entries);
         $this->assertEquals(['Post 2', 'Post 3'], $entries->map->title->all());
     }
+    
+    /** @test */
+    public function entries_retrieval_with_join_table()
```

Comment:
maybe rename it to something more descriptive like `entries_can_be_retrieved_and_are_not_null_when_using_join`